src/Core/AdminConsole/OrganizationFeatures/Policies/Implementations/SavePolicyCommand.cs

```diff
@@ -78,6 +80,23 @@ public async Task<Policy> SaveAsync(PolicyUpdate policyUpdate)
         return policy;
     }
 
+    public async Task<Policy> VNextSaveAsync(SavePolicyModel policyModel)
```

Comment:
I suggest a comment here, e.g.

> FIXME: this is a first pass at implementing side effects after the policy has been saved, which was not supported by the validator pattern. However, this needs to be implemented in a policy-agnostic way rather than building out switch statements in the command itself.

(my understanding at least; feel free to make it less opinionated as we're still going to discuss this as a team)